main.go

```diff
@@ -136,7 +137,7 @@ func NewCacheModel(configs *ConfigsModel) *CacheModel {
 
 	return &CacheModel{
 		PathList: splittedPaths,
-		//FilePathMap:         map[string]string{},
+		//FilePathMap:       map[string]string{},
```

Comment:
Please do not uncomment this line, can be used anywhere else, so this way we won't run into a nil map

Reply:
nem uncommenteltem, ket spacet vettem ki

Reply:
this is a public repo, please use english (in private ones too ;) )

Reply:
ops, sorry

Reply:
sry